### PR TITLE
Add URL display to weblink URLs

### LINKED ISSUE #3840: User-created WebLink details should display url in properties
We should display the url of the weblink when the catalog entity properties are opened in the catalog. At the moment user can't see the url at all.

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

fixes #3840

Screen Shot 2021-10-14 at 11 57 29 AM

### CODE REVIEW

File: src/renderer/initializers/catalog-entity-detail-registry.tsx [hunk 1]
```diff
@@ -45,6 +45,20 @@ export function initCatalogEntityDetailRegistry() {
...
+          Details: ({ entity }: CatalogEntityDetailsProps<WebLink>) => (
+            <>
+              <DrawerTitle title="More Information" />
+              <DrawerItem name="URL">
+                {entity.spec.url}
+              </DrawerItem>
+            </>
+          ),
+        },
+      },
```
Comment (aleksfront) [inline src/renderer/initializers/catalog-entity-detail-registry.tsx:62]: Ideally, this file shouldn't contain additional information for different kind of entities. Such details must be added through the specific entity files.

File: src/renderer/initializers/catalog-entity-detail-registry.tsx [hunk 2]
```diff
@@ -45,6 +45,20 @@ export function initCatalogEntityDetailRegistry() {
...
         },
-      }
+      },
+      {
+        apiVersions: [WebLink.apiVersion],
+        kind: WebLink.kind,
+        components: {
+          Details: ({ entity }: CatalogEntityDetailsProps<WebLink>) => (
+            <>
+              <DrawerTitle title="More Information" />
```
Comment (aleksfront) [inline src/renderer/initializers/catalog-entity-detail-registry.tsx:55]: Maybe just `More`?

**Review by aleksfront — APPROVED** [review]

File: src/renderer/initializers/catalog-entity-detail-registry.tsx (hunk 1 above)
Comment (Nokel81) [reply src/renderer/initializers/catalog-entity-detail-registry.tsx:62]: I don't understand, how is this different than, say, the file `src/renderer/initializers/kube-object-details-registry.tsx`?

File: src/renderer/initializers/catalog-entity-detail-registry.tsx (hunk 2 above)
Comment (Nokel81) [reply src/renderer/initializers/catalog-entity-detail-registry.tsx:55]: I don't like one word titles like that, though I also don't like the title I have now...

I would have wanted it to be in the same format as the lines above but couldn't get it to be. Probably a place for improvement in the future.

**Pull request merged**